samples/1-Login.md

```diff
@@ -0,0 +1,19 @@
+## Login
+Login with default client id & tenant id
+
+```sh
+$ mgc login --scopes User.ReadWrite Mail.ReadWrite
```

Comment:
QQ: why do you have the dollar sign? won't it make copy pasting for people harder?

Reply:
I've seen it as a convention in a lot of samples. I always found that copying and pasting was fine for me since I actually never used the copy button 😬
Would you suggest removing it?

Reply:
yes I would, from a simplicity perspective (C/P experience) and also to align with the public docs, see [this example](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli)